public/stylesheets/masters/snow/README.md

```diff
@@ -0,0 +1,7 @@
+# Snow Overview
+
+The Archive briefly used pale lavendar for its background color and several
+other accents. Snow is a legacy skin that turned those things pure white.
+
+Snow is still used as a parent for Snow Blue because several elements it turns
```

Comment:
"because it turns several elements from pale grey in the default skin to pure white"?

Reply:
"Because some of the elements that Snow turns pure white were changed to pale grey in the default skin, it is necessary to keep using Snow as a parent for Snow Blue"?

Reply:
:+1: